Code/GraphMol/Descriptors/DCLV.h

```diff
@@ -0,0 +1,86 @@
+#ifndef RDKIT_DCLV_H
```

Comment:
I forgot this in my other suggestions, but please add a copyright statement modelled on other RDKit files to this and DCLV.cpp